owners/src/parser.js

```diff
@@ -164,53 +164,64 @@ class OwnersParser {
    *     rule.
    */
   _parseOwnersDict(ownersPath, ownersDict) {
-    let [[pattern, ownersList]] = Object.entries(ownersDict);
+    const [[fullPattern, ownersList]] = Object.entries(ownersDict);
     const rules = [];
     const errors = [];
+    let patternList = [];
```

Comment:
Don't let the diff fool you; all of the parsing logic is just indented into a `forEach` loop for each subpattern, the same way it was before #437 .

Reply:
Pro-tip: Adding `?w=1` to the diff URL hides all whitespace-only changes.

https://github.com/ampproject/amp-github-apps/pull/448/files?w=1